front/app/components/ProjectCard/index.tsx

```diff
@@ -445,14 +448,17 @@ const ProjectCard = memo<InputProps>(
 
     const handleProjectCardOnClick = (projectId: string) => () => {
       trackEventByName(tracks.clickOnProjectCard, { extra: { projectId } });
+      scrollToTop();
```

Comment:
In cases like these, let's handle the scroll behaviour where the actual navigation happens, in this case in the `styled(Link)` container.